python/gen_requirements.py

```diff
@@ -143,6 +143,7 @@
         (
             "Requirements for the tvmc command-line tool",
             [
+                "ethos-u-vela",
```

Comment:
Does it also need to be added to the `CONSTRAINTS` list? Comment on line 218 says:

`# 1. Each package specified in REQUIREMENTS_BY_PIECE must be included here.`

Reply:
It is there already. I’m just adding it in the tvmc dependency set.